### ISSUE TITLE
Batch_size in fit_generator()

### ISSUE THREAD

**Issue by saeedalahmari3** [keras-team/keras#8502]
steps per epochs in documentations mentioned batch_size in order to choose the steps_per_epoch. However, what is the default batch_size when using fit_generator()? I know that using fit() batch_size default is 32. and this is hyper-parameter that needs to be tuned since it involves in gradient updates.
Now how to tune batch_size when using fit_generator()? Help please.

**Comment by brge17** [comment]
The generator, which you either write your own or use the ones included with Keras, is supposed to yield data of the appropriate batch_size.

As per such it is a parameter of the generator itself, and not fit generator.

**Comment by ozabluda** [comment]
Recent changes to the docs in the repository make it somewhat clearer:
https://github.com/fchollet/keras/blob/master/keras/engine/training.py#L1929
